plugins/index.js

```diff
@@ -90,8 +91,16 @@ export function loadPlugin(manifest) {
         script.src = getSiteURL() + bundlePath;
         script.onload = onLoad;
         console.log('Loading ' + manifest.id + ' plugin'); //eslint-disable-line no-console
-        document.getElementsByTagName('head')[0].appendChild(script);
 
+        if (!tryRequire(bundlePath)) {
+            console.error('Could not load bundle for ' + manifest.id); //eslint-disable-line no-console
+
+            loadedPlugins[manifest.id] = false;
```

Comment:
Wondering if clients might be checking the existence of a key vs. testing the value -- should we just skip setting this altogether?

Reply:
✅